src/scripts/main.js

```diff
@@ -1,3 +1,417 @@
+officeForForm.appendChild(singapore);
+officeForForm.appendChild(london);
+officeForForm.appendChild(newYork);
+officeForForm.appendChild(edinburgh);
+officeForForm.appendChild(sanFrancisco);
+
+const positionForForm = document.createElement('input');
+
+positionForForm.setAttribute('type', 'text');
+positionForForm.setAttribute('name', 'position');
+positionForForm.setAttribute('required', '');
+positionForForm.dataset.qa = 'position';
+
+const salaryForForm = document.createElement('input');
+
+salaryForForm.dataset.qa = 'salary';
+salaryForForm.setAttribute('type', 'number');
+salaryForForm.setAttribute('name', 'salary');
+salaryForForm.setAttribute('required', '');
+
+const submit = document.createElement('button');
+
+submit.setAttribute('type', 'submit');
+submit.textContent = 'Save to table';
+
+const label1 = document.createElement('label');
+
+label1.textContent = 'Name: ';
+label1.appendChild(nameForForm);
+
+const label2 = document.createElement('label');
+
+label2.textContent = 'Position: ';
+label2.appendChild(positionForForm);
+
+const label3 = document.createElement('label');
+
+label3.textContent = 'Age: ';
+label3.appendChild(ageForForm);
+
+const label4 = document.createElement('label');
+
+label4.textContent = 'Salary ';
+label4.appendChild(salaryForForm);
+
+const labelOffice = document.createElement('label');
+
+labelOffice.textContent = 'Office: ';
+labelOffice.appendChild(officeForForm);
+
+form.appendChild(label1);
+form.appendChild(label2);
+form.appendChild(labelOffice);
+form.appendChild(label3);
+form.appendChild(label4);
+form.appendChild(submit);
+
+const pushNotification = (posTop, posRight, title, description, type) => {
+  const notification = document.createElement('div');
+
+  document.body.appendChild(notification);
+  notification.classList.add('notification');
+  notification.dataset.qa = 'notification';
+
+  const h2 = document.createElement('h2');
+
+  h2.textContent = title;
+  notification.appendChild(h2);
+
+  const about = document.createElement('p');
+
+  about.textContent = description;
+  notification.appendChild(about);
+
+  if (type === 'warning') {
+    notification.classList.add('warning');
+  }
+
+  if (type === 'error') {
+    notification.classList.add('error');
+  }
+
+  if (type === 'success') {
+    notification.classList.add('success');
+  }
+
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  setTimeout(() => {
+    notification.remove();
+  }, 5000);
+};
+
+submit.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const row = document.querySelector('tr').cloneNode(true);
+  const userName = nameForForm.value;
+
+  const userPosition = positionForForm.value;
+  const userOffice = officeForForm.value;
+  const userAge = ageForForm.value;
+  const salaryInput = salaryForForm.value.toString();
+  const sal = salaryInput.split('').reverse();
+  const formattedZarplatnya = [];
```

Comment:
The logic for formatting the salary seems incorrect. The salary is split into characters and reversed, which might not format the salary correctly. Consider revising this logic to ensure proper currency formatting.